commands/scripts.py

```diff
@@ -11,7 +11,10 @@
 
 from stat import ST_MODE
 from setuptools import Command
-from distutils.dep_util import newer
+try:
+    from distutils.modified import newer
```

Comment:
This is using distutils still, should this be setuptools also?

Reply:
Done.

Reply:
And yes, it should :)